src/book.py

```diff
@@ -824,13 +981,48 @@ def detect_exchange(self, file_path: Path) -> Optional[str]:
                     "ledgers",
                 ],
                 "bitpanda_pro_trades": [
-                    "Disclaimer: All data is without guarantee,"
-                    " errors and changes are reserved."
+                    "Order ID",
+                    "Trade ID",
+                    "Type",
+                    "Market",
+                    "Amount",
+                    "Amount Currency",
+                    "Price",
+                    "Price Currency",
+                    "Fee",
+                    "Fee Currency",
+                    "Time (UTC)",
+                ],
+                "bitpanda": [
+                    "Transaction ID",
+                    "Timestamp",
+                    "Transaction Type",
+                    "In/Out",
+                    "Amount Fiat",
+                    "Fiat",
+                    "Amount Asset",
+                    "Asset",
+                    "Asset market price",
+                    "Asset market price currency",
+                    "Asset class",
+                    "Product ID",
+                    "Fee",
+                    "Fee asset",
+                    "Spread",
+                    "Spread Currency",
                 ],
             }
-            for exchange, expected in expected_headers.items():
-                if header == expected:
-                    return exchange
+            with open(file_path, encoding="utf8") as f:
+                reader = csv.reader(f)
+                # check all potential headers at their exprected header row,
+                # rewind the file after each expected header checked
+                for exchange, expected in expected_headers.items():
+                    header_row_num = expected_header_row[exchange]
+                    for _ in range(header_row_num + 1):
+                        header = next(reader)
```

Comment:
This raises an error for empty files or files which have less lines than `header_row_num`. Use `next(reader, None)` instead.